src/Sql/Sql.Test/ScenarioTests/ManagedDatabaseLogReplayScenarioTest.cs

```diff
@@ -38,35 +38,35 @@ public ManagedDatabaseLogReplayScenarioTest(ITestOutputHelper output) : base(out
             };
         }
 
-        [Fact]
+        [Fact(Skip = "Depends on hardcoded resource to rerecord")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestManagedDatabaseLogReplayService()
         {
             RunPowerShellTest("Test-ManagedDatabaseLogReplay");
         }
 
-        [Fact]
+        [Fact(Skip = "Depends on hardcoded resource to rerecord")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestCompleteManagedDatabaseLogReplayService()
         {
             RunPowerShellTest("Test-CompleteManagedDatabaseLogReplay");
         }
 
-        [Fact]
+        [Fact(Skip = "Depends on hardcoded resource to rerecord")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestCancelManagedDatabaseLogReplayService()
         {
             RunPowerShellTest("Test-CancelManagedDatabaseLogReplay");
         }
 
-        [Fact]
+        [Fact(Skip = "Depends on hardcoded resource to rerecord")]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestPipingManagedDatabaseLogReplayService()
         {
             RunPowerShellTest("Test-ManagedDatabaseLogReplayPiping");
         }
 
-        [Fact]
+        [Fact(Skip = "Depends on hardcoded resource to rerecord")]
```

Comment:
We need to ge a positive confirmation from the test owner (or other knowledgeable dev) that this behavior has not regressed.

Reply:
Signing off on this test - @MilanBrkicFON will follow up to make the test not depend on hardcoded resources.

Reply:
Is this for all the tests in this file, or just this specific test?

Reply:
The whole file.